externs/shaka/player.js

```diff
@@ -795,6 +796,10 @@ shaka.extern.ManifestConfiguration;
  *   of a segment. This affects both explicit start times and calculated start
  *   time for live streams. This can put us further from the live edge. Defaults
  *   to <code>false</code>.
+ * @property {number} gapDetectionThreshold
+ *   The limit (in seconds) for a gap  gap detection threshold. This value
```

Comment:
I would suggest wording this in terms of what it does.  Something like `The maximum distance (in seconds) before a gap when we'll automatically jump.`

Reply:
Done!